### DISCUSSION TITLE
How to broadcast notification to the bell icon? (notify admin users)

### DISCUSSION THREAD

**Discussion by DevMushref** [filamentphp/filament#5421] [Help] [answered]
I checked all possible Q&A here that might be related to my question, but didn't find.

I followed the broadcasting docs. In my model I wrote:

```
public function delete()
     {
         parent::delete();

         $recipient = auth()->user();

        Notification::make()
            ->title('Support deleted!')
            ->broadcast($recipient)
            ->send();
     }
```

It works, but as a popup notification on the screen, not to the bell icon (notifications table). Furthermore, How may I send a notification after a record is created to admin users? (what's the method/function name if available)?

Thank you

**Comment by danharrin** [comment, marked as answer]
broadcasting is not the same as database notifications

https://github.com/filamentphp/demo/blob/main/app/Filament/Resources/Shop/OrderResource/Pages/CreateOrder.php#L19-L32

**Reply by DevMushref** [reply]
Dear dan, how can I use the `hasRole()` method to send notifications to the desired roles only?

I tried `User::get()->hasRole()` and `->toDatabase(auth()->user()->hasRole('security_supervisor'));`
